README.md

```diff
@@ -60,33 +60,39 @@ The Proxies have to fetch certificates from the central Certificate Authority, h
 
 ## Getting started
 
+If you have docker installed and cloned our repository you can run a demo beam network using the following command:
+```bash
+./dev/beamdev demo
+```
+This will launch a beam network consisting of two proxies (Proxy1 localhost:8081 and Proxy2 localhost:8082) and a broker (localhost:8080)
+
 The following paragraph simulates the creation and the completion of a task
 using [cURL](http://curl.se) calls. Two parties (and their Samply.Proxies) are
-connected via a central broker. Each party runs an application, called `app`.
-We will simulate this application.
+connected via a central broker. Each party has one registered application.
+In the next section we will simulate the communcation between these applications over the beam network.
 
-Note: cURL versions before 7.82 do not support the `--json` option. In this case, please use `--data` instead.
+> Note: cURL versions before 7.82 do not support the `--json` option. In this case, please use `--data` instead.
 
 The used BeamIds are the following:
 
 | System             | BeamID                       |
 |--------------------|------------------------------|
-| Broker             | broker.example.de            |
-| Proxy1             | proxy1.broker.example.de     |
-| App behind Proxy 1 | app.proxy1.broker.example.de |
-| Proxy2             | proxy2.broker.example.de     |
-| App behind Proxy 2 | app.proxy2.broker.example.de |
+| Broker             | broker                       |
+| Proxy1             | proxy1.broker                |
+| App behind Proxy 1 | app1.proxy1.broker           |
+| Proxy2             | proxy2.broker                |
+| App behind Proxy 2 | app2.proxy2.broker           |
 
-In this example, we use the same ApiKey `AppKey` for both parties.
+In this example, we use the same ApiKey `App1Secret` for both parties.
```

Comment:
```suggestion
To simplify this example, we use the same ApiKey `App1Secret` for both apps. Also, the Broker has a short name (`broker`) where in a real setup, it would be required to have a fully-qualified domain name as `broker1.samply.de` (see [System Architecture](#system-architecture)).
```